packages/ui-extensions/src/surfaces/checkout/api/shipping/shipping-option-list.ts

```diff
@@ -0,0 +1,6 @@
+export interface ShippingOptionListApi {
+  /**
+   * The delivery group id the item is attached to
```

Comment:
Do we usually use `item`? It doesn't feel super description. Maybe: "The unique identifier of the delivery group the extension is referencing"?